docs/migration/dapp_migration.md

```diff
@@ -48,4 +49,4 @@ For starters, here's a sampling of exciting perks dapps with get with the Nitro
 
 - **Gas Info in Transaction Receipts**: Arbitrum transaction receipts return data about gas in a new format; receipts will have `gasUsed` (total) and `gasUsedForL1` fields (instead of the `feeStats` field in Arbitrum classic).
 
-- **Batch Info In Receipts**: Arbitrum transaction receipts no longer include the `l1SequenceNumber` field; the `findBatchContainingBlock` method in the [NodeInterface](../../contracts/src/node-interface/NodeInterface.sol) virtual precompile can be used to determine a transaction's inclusion in L1.
\ No newline at end of file
+- **Batch Info In Receipts**: Arbitrum transaction receipts no longer include the `l1SequenceNumber` field; the `findBatchContainingBlock` or `getL1Confirmations` methods in the [NodeInterface precompile](../arbos/Precompiles.md) can be used to determine a transaction's inclusion in L1.
```

Comment:
The node interface isn't a precompile (as it can't be accessed from the EVM), and therefore isn't in this list. The previous link should work.